service/header/sync.go

```diff
@@ -57,37 +61,8 @@ func (s *syncer) Sync(ctx context.Context) {
 	}
 }
 
-// TODO(@Wondertan): Number of headers that can be requested at once. Either make this configurable or,
-// find a proper rationale for constant.
-var requestSize uint64 = 128
-
-// syncDiff requests headers from knownHead up to new head.
-func (s *syncer) syncDiff(ctx context.Context, knownHead, newHead *ExtendedHeader) error {
-	start, end := uint64(knownHead.Height+1), uint64(newHead.Height)
-	for start < end {
-		amount := end - start
-		if amount > requestSize {
-			amount = requestSize
-		}
-
-		headers, err := s.exchange.RequestHeaders(ctx, start, amount)
-		if err != nil {
-			return err
-		}
-
-		err = s.store.Append(ctx, headers...)
-		if err != nil {
-			return err
-		}
-
-		start += amount
-	}
-
-	return s.store.Append(ctx, newHead)
-}
```

Comment:
~~This is not necessary anymore because we assume that this will always start from genesis now?~~ NVM

Reply:
K